qCC/ccGraphicalSegmentationTool.cpp

```diff
@@ -903,7 +903,6 @@ void ccGraphicalSegmentationTool::segment(bool keepPointsInside, ScalarType clas
 
 		// we project each point and we check if it falls inside the segmentation polyline
 #if defined(_OPENMP)
-		omp_set_num_threads(std::max(1, omp_get_max_threads() - 1)); // always leave one thread/core to let the application breath
```

Comment:
Is removing this line intentional ?

Reply:
Yes, it's not worth the sweat to do this for some short operations (and on some processors with a few cores, it makes the computation a lot slower). I should probably add a minimum to the number of threads used... Let me fix that (again)